<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviseView</class>
 <widget class="QWidget" name="ReviseView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Enter definition for:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="keyLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="textInput"/>
   </item>
   <item>
    <widget class="QPushButton" name="checkButton">
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="homeButton">
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
